sunbird_service_api_test/src/test/java/org/sunbird/integration/test/course/batch/AddUserToCourseBatch.java

```diff
@@ -0,0 +1,115 @@
+package org.sunbird.integration.test.course.batch;
+
+import com.consol.citrus.annotations.CitrusTest;
+import com.consol.citrus.testng.CitrusParameters;
+import java.util.UUID;
+import javax.ws.rs.core.MediaType;
+import org.springframework.http.HttpStatus;
+import org.sunbird.common.action.ContentStoreUtil;
+import org.sunbird.common.action.CourseBatchUtil;
+import org.sunbird.integration.test.common.BaseCitrusTestRunner;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class AddUserToCourseBatch extends BaseCitrusTestRunner {
+
+  public static final String TEST_NAME_ADD_USER_TO_BATCH_FAILURE_WITH_INVALID_BATCHID =
+      "testAddUserToBatchFailureWithInvalidBatchId";
```

Comment:
Rename constants and test names as follows:

testAddUserToCourseBatchFailureWithInvalidBatchId
testAddUserToCourseBatchFailureWithoutAuthToken
testAddUserToCourseBatchFailureWithoutUserIds
testAddUserToCourseBatchFailureForOpenBatch
testAddUserToCourseBatchFailureForInviteOnlyBatchWithoutOrg